magicctapipe/scripts/lst1_magic/semi_automatic_scripts/config_general.yaml

```diff
@@ -0,0 +1,42 @@
+mc_tel_ids:
+    LST-1: 1
+    LST-2: 0
+    LST-3: 0
+    LST-4: 0
+    MAGIC-I: 2
+    MAGIC-II: 3
+
+directories:
+    workspace_dir : "/fefs/aswg/workspace/elisa.visentin/auto_MCP_PR/"  # Output directory where all the data products will be saved.
+    # MC paths below are ignored if you set NSB_matching = true.
+    MC_gammas     : "/fefs/aswg/data/mc/DL0/LSTProd2/TestDataset/sim_telarray"  # set to "" if you don't want to process these Monte Carlo simulations.
+    MC_electrons  : "" 
+    MC_helium     : "" 
+    MC_protons    : "/fefs/aswg/data/mc/DL0/LSTProd2/TrainingDataset/Protons/dec_2276/sim_telarray" 
+    MC_gammadiff  : "/fefs/aswg/data/mc/DL0/LSTProd2/TrainingDataset/GammaDiffuse/dec_2276/sim_telarray/"
+    
+data_selection:
+    source_name_database: "CrabNebula"  # MUST BE THE SAME AS IN THE DATABASE; Set to null to process all sources in the given time range.
+    source_name_output: 'Crabtest'  # Name tag of your target. Used only if source_name_database != null.
+    target_RA_deg : 83.629  # RA in degrees; Set to null if source_name_database=null.
+    target_Dec_deg: 22.015  # Dec in degrees; set to null if source_name_database=null.
+    time_range : True  # Search for all runs in a LST time range (e.g., 2020_01_01 -> 2022_01_01).
+    min : "2023_11_17"
+    max : "2024_03_03"   
+    date_list : ['2020_12_15','2021_03_11']  # LST list of days to be processed (only if time_range=False), format: YYYY_MM_DD.
+    skip_LST_runs: [3216,3217]  # LST runs to ignore.
+    skip_MAGIC_runs: [5094658]  # MAGIC runs to ignore.
+    
+general:
+    SimTel_version: "v1.4"  
+    LST_version   : "v0.10"
```

Comment:
I've noticed that on my computer, the script list_from_h5 works correctly only with version 0.9 LST_version in config file. When I switch to version 0.10, the script does not seem to locate the LST runs. I suspect the issue might be related to the observations_LST.h5 file, specifically the lstchain-version noted in the 8th column, which remains set to v0.9.

Reply:
Thank you. I will check the LST database (there were some bugs in the related scripts and maybe I still get the wrong version). For sure the list_from_h5 scripts check if the version requested in the config_general is the same of the processed file for NSB (i.e., generally the last available version for a run), so in case of bugs in file versions the files cannot be found. I will check it hopefully today/tomorrow

Reply:
I didn't use config_general with v0.10. I have just taken another look at your table and you're right that some rows are labeled v0.10 and others v0.9. I worked with data from March 2022, and it took me some time to figure out why list_from_h5 couldn't find the LST runs. Therefore, I think we should add a brief comment in the config file reminding users to check the LST version.

Reply:
I will add it (also in the Readme). Thanks

Reply:
Actually, the script that writes the lists check for the lstchain_processed_file version (i.e., last version), i.e., the one used to evaluate the NSB: in case of no errors, the NSB value should be the same for 0.9 and 0.10 if both are present and it is always better to use the last available version; in case of errors (e.g., missing flatfield), we discard the run if the NSB evaluation fails and we don't propagate the errors to the other steps.
So, you list the run with 0.10 version if you use 0.10 in the config AND if the file processed to evaluate the NSB is the 0.10 one